providers/rbs/converter/converter.go

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package converter
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/facebookincubator/nvdtools/cvefeed/nvdcommon"
+	"github.com/facebookincubator/nvdtools/wfn"
+
+	dstSchema "github.com/facebookincubator/nvdtools/cvefeed/jsonschema"
+	srcSchema "github.com/facebookincubator/nvdtools/providers/rbs/schema"
+)
+
+const (
+	cveDataVersion = "4.0"
+	timeLayout     = "2006-01-02T15:04:05Z"
+)
+
+// Convert converts iDefense vulnerability to NVD format
+func Convert(input <-chan *srcSchema.Vulnerability) *dstSchema.NVDCVEFeedJSON10 {
+	var feed dstSchema.NVDCVEFeedJSON10
+	for vuln := range input {
+		converted, err := convert(vuln)
+		if err != nil {
+			log.Println(err)
+			continue
+		}
+		feed.CVEItems = append(feed.CVEItems, converted)
+	}
+	return &feed
+}
+
+func convert(item *srcSchema.Vulnerability) (*dstSchema.NVDCVEFeedJSON10DefCVEItem, error) {
```

Comment:
I'd make it a method of `srcSchema.Vulnerability` and ditched the `Convert` above -- it feels like something that whaterver sets up the input channel should do.

Reply:
Yeah, it's going to be like that in the next diffs, unifying all converters. For now, it does the same thing as others do